rsmtool/notebooks/header.ipynb

```diff
@@ -202,80 +202,84 @@
    "outputs": [],
    "source": [
     "# Read in the training and testing features, both raw and pre-processed\n",
-    "# Make sure that the `spkitemid` column is read as a string\n",
+    "# Make sure that the `spkitemid` and `candidate` columns are read as strings \n",
+    "# to preserve any leading zeros\n",
     "# We filter DtypeWarnings that pop up mostly in very large files\n",
     "\n",
+    "string_columns = ['spkitemid', 'candidate']\n",
+    "converter_dict = dict([(column, str) for column in string_columns if column])\n",
```

Comment:
You can use a dictionary comprehension which will be much more readable

`{column: str for column in string_columns}`